tokio/src/task/blocking.rs

```diff
@@ -114,6 +114,20 @@ cfg_blocking! {
         F: FnOnce() -> R + Send + 'static,
         R: Send + 'static,
     {
+        #[cfg(all(feature = "tracing", tokio_unstable))]
+        #[cfg_attr(docsrs, doc(cfg(all(feature = tokio_unstable))))]
+        let f = {
+            let span = tracing::trace_span!(
+                target: "tokio::task",
+                "task",
+                kind = %"blocking",
+                function = %std::any::type_name::<F>(),
+            );
+            move || {
+                let _g = span.enter();
+                f()
+            }
+        };
```

Comment:
I _think_ this will mean that the time between when `spawn_blocking` is called, and when the actual closure is invoked, won't be measured?

Reply:
It should be measured. The span is created immediately when `spawn_blocking` is called, but only entered when the closure is invoked. Thus, the difference between the span’s total duration and its active duration is the time the closure was waiting to be invoked.